freebsd/FreeBSDProcessList.c

```diff
@@ -494,10 +494,6 @@ void ProcessList_goThroughEntries(ProcessList* this) {
                proc->percent_cpu = proc->percent_cpu / (double) cpus;
          }
       }
-      if (isIdleProcess == false && proc->percent_cpu >= 99.8) {
-         // don't break formatting
-         proc->percent_cpu = 99.8;
-      }
```

Comment:
Oh sorry, I misread the diff and thought you were adding this instead of removing. It makes total sense. I probably need more coffee. :)